src/platform/OpenThread/GenericThreadStackManagerImpl_OpenThread.cpp

```diff
@@ -866,6 +866,13 @@ CHIP_ERROR GenericThreadStackManagerImpl_OpenThread<ImplClass>::_GetExternalIPv6
     return CHIP_DEVICE_ERROR_CONFIG_NOT_FOUND;
 }
 
+template <class ImplClass>
+CHIP_ERROR GenericThreadStackManagerImpl_OpenThread<ImplClass>::_GetPollPeriod(uint32_t & buf)
```

Comment:
Seems like there already exist `SetThreadPollingConfig` and `GetThreadPollingConfig`. You may consider using them instead of adding a new method (although looks like `mPollingConfig` is not initialized with OT default values, so it may require some changes). If you prefer keeping the `GetPollPeriod` method, please add thread stack locking/unlocking.

Reply:
Added locking and unlocking Thread stack